VM/rel2vm_params.py

```diff
@@ -559,6 +558,33 @@ def centre_lon(lat_target: float):
     return lon_target
 
 
+def write_srf_path(srf_corners: np.ndarray, wd: Path):
+    """
+    Write a temporary file srf.path containing SRF corner coordinates.
+    Used for plotting SRF planes using GMT
+
+
+
+    Parameters
+    ----------
+    srf_corners : corners of the srf
+    wd : working directory
+
+    Returns
+    -------
+    path to the srf path file
+
+    """
+    srf_path = wd / "srf.path"
+    if not srf_path.exists():
+        with open(srf_path, "wb") as sp:
+            for plane in srf_corners:
+                sp.write("> srf plane\n".encode())
+                np.savetxt(sp, plane, fmt="%f")
+                sp.write("%f %f\n".encode() % (tuple(plane[0])))
```

Comment:
maybe just use an f string. Also, why open the file in byte mode if you're saving just text and an array as text?

Reply:
Just copied the old code without extra attempt to modernise it :)